src/tests/mocks/template-default-mocks.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 
 // eslint-disable-next-line no-undef
-jest.mock('../../components/Templates/TemplateDefault', () => {
-  // eslint-disable-next-line react/prop-types
-  return function PageCardsEdit({ children }) {
+jest.mock('../../components/Templates/TemplateDefault', () =>
+  // eslint-disable-next-line react/prop-types,implicit-arrow-linebreak
```

Comment:
Don't think you should be disabling this like that.  Please remove the line ignore